### DISCUSSION TITLE
locus relations equivalent to

### DISCUSSION THREAD

**Discussion by lbestock** [arch-kiosk/arch-kiosk-office#1465] [users: ideas for new features]
Think we've discussed in the past. Important for interpretation is when two loci end up being the same thing. Problem is when it's likely, and thus important for interpretation, but not certain. I'm running into it now with FA Sondages - an organicy construction layer found in two Sondages is probably the same layer. This is not a small issue really - I have four programs open including FM on two devices, and a piece of paper, and I am having trouble figuring out things that the system could help with a lot in analysis, but for this piece of it the recording would have had to accommodate it first. My instinct is to say make an 'equivalent to' relation and excavators will have to say in interpretation that it is probable not certain. A bigger task would be giving the ability to flag a relation as probable but not certain.

**Comment by urapadmin** [comment]
I can easily introduce a new value "equivalent" to. But there is more here and we should talk about what kind of analysis you would need.

**Comment by urapadmin** [comment]
is that related to #614 ?

**Comment by lbestock** [comment]
Kinda. In the sense that loci which are equivalent (and I so wanted this recently when writing) are necessarily the same phase. The sondages remain the best example, and because it was done inconsistently in the field - in two sondages the presumed same layers were sometimes made different lots of the same locus and sometimes made different loci. Most properly they should be made different loci with the presumption of equivalence markable. The question remains if that is truly a locus relationship or a different kind of link between the two. The problem also remains that there are some cases where it is presumed, like with apparently equivalent strata in two different sondages, and some cases where it will have been proved and we can say entirely: they are equivalent and should never be separated for analysis (as will be the case when I remove the area between the two sondages and show that no wall below the floor separates them, they really are continuous strata present in both holes).

**Comment by urapadmin** [comment]
I can't see a solution emerging from that. Just introducing equivalence as a relation is clearly not it.

**Comment by lbestock** [comment]
#613 and #614 were this again, with the different unit/same unit issue helping show why a locus relation is not the right solution.

Test case to discuss in Laurel's preferred method of an actual example: PVD2020 units LA and LB share a wall. It's context LA-004. What should it be in LB?

**Comment by urapadmin** [comment]
a locus relation equivalent to would have no implications for the software. But it seemed that you guys want to have something that closely links two loci so that they are more or less the same locus. 

I suggested not to do that with a relation (arguing archaeologically that this is not a locus relationship since it is arbitrarily created by the excavation - like a unit).I wanted to merge two loci if they are equivalent, you did not like it. The discussion ended aporetically. 

I also suggested to have a special new attribute that links loci technically (as in archaeologically) but I still don't know what implications are expected. So that is what we should find out:
What are the implications of locus equivalence? ->

**Comment by urapadmin** [comment]
The topic flared up again: #1790
